sale_layout_category_hide_detail/static/src/css/boolean_fa_icon_widget.scss

```diff
@@ -0,0 +1,35 @@
+div.fa-icon-widget > input.form-check-input,
```

Comment:
Why not using bootstrap classes in the component template to avoid as much css as possible?